orango.js.org

roboncode opened the pull request:
- There is reasonable content on the page (see: [No Content](https://github.com/js-org/js.org/wiki/No-Content))
- I have read and accepted the [Terms and Conditions](http://js.org/terms.html)

indus commented:
I've already added your requested subdomain to JS.ORGs zonefile. Please make sure you fix the path problem so I can have a look at your page.
I'll merge after we have "contact".

roboncode commented:
Thx. It is now fixed. Rob

On Wed, Aug 29, 2018 at 3:31 AM Stefan Keim <notifications@github.com>
wrote:

> I've already added your requested subdomain to JS.ORGs zonefile. Please
> make sure you fix the path problem so I can have a look at your page.
> I'll merge after we have "contact".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/js-org/js.org/pull/2279#issuecomment-416889067>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AAyF8xCcnQJKk-TU5TigMXRsj6qhH2ROks5uVl-JgaJpZM4WRFCG>
> .
>

indus commented:
Welcome https://orango.js.org
